pages/first_page.py

```diff
@@ -27,19 +27,16 @@
   else:
     st.write("Number of inputs in table will be: {}".format(mtotal))
 
-  # Stream inputs from the app
-  all_inputs = []
-  for inp in lister.inputs_generator():
-    all_inputs.append(inp)
-
-    if len(all_inputs) >= mtotal:
-      break
+  # Stream inputs from the app. list_inputs give list of dictionaries with inputs and its metadata .
+  input_obj = User(user_id=userDataObject.user_id).app(app_id=userDataObject.app_id).inputs()  
```

Comment:
As this is a main template page, could we do something different than listing inputs maybe `Image Generation`?!
@sriram-clarifai your inputs please.

Reply:
@mogith-pn We should limit all_inputs by `mtotal` variable at here
https://github.com/Clarifai/module-template/blob/9197eb865f0a12547b31130f3721dfc341857569/pages/first_page.py#L36

Reply:
@sriram-clarifai If no further comments, we'll proceed with existing `list inputs` functionality in model template page